TTHAnalysis/python/tools/higgsDiffCompTTH.py

```diff
@@ -233,7 +306,10 @@ def analyze(self, event):
             # Compare the matched jets and the reco jets
             # First I want to write down a few quantities
             visHiggs_matched, visHiggsPlusNu_matched = self.makeVisibleHiggs(leptonFromHiggs, jm1, jm2, NuFromWFromH[0] if len(NuFromWFromH)==1 else None)
-
```

Comment:
Please don't just remove empty lines , otherwise merging with central won't be approved ever